src/domain/collection/curatedEvents/__tests__/CuratedEvents.test.tsx

```diff
@@ -26,21 +37,32 @@ const collection = fakeCollection({
   curatedEvents,
 }) as CollectionFieldsFragment;
 
-const getMocks = (events: EventFieldsFragment[], ids = eventIds) => [
+const getMocks = (
```

Comment:
One more valid test case could be that both events and courses list can be present at the same time? But not sure if it's worth of investment...

Reply:
added course to "should show all events" test